src/java.desktop/share/classes/java/awt/Menu.java

```diff
@@ -486,24 +487,24 @@ MenuItem getShortcutMenuItem(MenuShortcut s) {
         return null;
     }
 
-    synchronized Enumeration<MenuShortcut> shortcuts() {
```

Comment:
I'm not sure whether it's ok to change return type here, probably it'd be better to keep `Enumeration` and use `return Collections.enumeration(shortcuts);` in the last line

Reply:
It's not a public API. As I see from other PR/commits changing package-private methods shouldn't a problem.

Reply:
Even non-public method can be called via reflection, so I'd be cautios about changing return type

Reply:
reverted